CONTRIBUTORS.md

```diff
@@ -57,6 +57,7 @@ This file lists the contributors to the PureScript compiler project, and the ter
 - [@philopon](https://github.com/philopon) (Hirotomo Moriwaki) - My existing contributions and all future contributions until further notice are Copyright Hirotomo Moriwaki, and are licensed to the owners and users of the PureScript compiler project under the terms of the [MIT license](http://opensource.org/licenses/MIT).
 - [@pseudonom](https://github.com/pseudonom) (Eric Easley) My existing contributions and all future contributions until further notice are Copyright Eric Easley, and are licensed to the owners and users of the PureScript compiler project under the terms of the [MIT license](http://opensource.org/licenses/MIT).
 - [@puffnfresh](https://github.com/puffnfresh) (Brian McKenna) All contributions I made during June 2015 were during employment at [SlamData, Inc.](#companies) who owns the copyright. I assign copyright of all my personal contributions before June 2015 to the owners of the PureScript compiler.
+- [@rightfold](https://github.com/rightfold) (rightfold) My existing contributions and all future contributions until further notice are Copyright rightfold, and are licensed to the owners and users of the PureScript compiler project under the terms of the [BSD 3-Clause license](https://opensource.org/licenses/BSD-3-Clause).
```

Comment:
I think we need to figure out how this will work, since I don't think BSD3 code can be redistributed under MIT, due to the additional clause about no advertising. I'd be fine to switch to BSD3 personally, which would be fine since MIT code can definitely be redistributed under BSD3. But we should discuss it.